Overview.src.html

```diff
@@ -1888,10 +1924,59 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   for HTTP/2, and equivalent information used to prioritize dispatch and processing of
   HTTP/1 fetches.
 
+
+ <li>
+  <p>If <var>request</var> is a <span>navigation request</span>, a user agent should, for each
+  <span title=concept-header>header</span> <span title=concept-header-name>name</span>
+  (<var>hint-name</var>) in the first column of the following table, if <var>hint-name</var>
+  is not in <var>request</var>'s <span title=concept-request-header-list>header list</span>,
+  <span title=concept-header-list-append>append</span>
+  <var>hint-name</var>/<var>hint-value</var> given in the same row on the second column, to
+  <var>request</var>'s <span title=concept-request-header-list>header list</span>.
+
+  <table>
+  <tr>
+   <th><span title=hint-name>hint-name</span>
+   <th><span title=hint-value>hint-value</span>
+  <tr>
+   <td>`<code title=http-dpr>dpr</a></code>`
+   <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#dpr>dpr value</a>
+  <tr>
+   <td>`<code title=http-save-data>save-data</code>`
+   <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#save-data>save-data value</a>
+  <tr>
+   <td>`<code title=http-viewport-width>viewport-width</code>`
+   <td>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#viewport-width>viewport-width value</a>
+  </table>
+
  <li>
   <p>If <var>request</var> is a <span>subresource request</span>, run these substeps:
 
   <ol>
+   <li><p>If the <var>request</var>'s <span title=concept-client-hint-list>client hints list</span> is not empty, run these substeps for each <var>hint-name</var> in the list:
+
+    <ol>
+     <li>
+      <p>Set <var>value</var> to the first matching statement, if any, switching on
+      <var>hint-name</var>:
+
+       <dl class=switch>
+        <dt>"<code title>dpr</code>"
+        <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#dpr>dpr value</a>
+        <dt>"<code title>save-data</code>"
+        <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#save-data>save-data value</a>
+        <dt>"<code title>viewport-width</code>"
+        <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#viewport-width>viewport-width value</a>
+        <dt>"<code title>width</code>"
+        <dd>a suitable <a href=http://httpwg.org/http-extensions/client-hints.html#width>width value</a></code>
+       </dl>
+
+     <li>
+      <p><span title=concept-header-list-append>Append</span>
```

Comment:
This `<p>` doesn't need to be on a newline since it's the only child of the `<li>`.